src/schema/v2/recentlySoldArtworks.ts

```diff
@@ -0,0 +1,173 @@
+import { GraphQLFieldConfig, GraphQLObjectType, GraphQLString } from "graphql"
+import { connectionDefinitions, connectionFromArray } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import { ArtworkType } from "./artwork"
+
+const RecentlySoldArtworkType = new GraphQLObjectType<any, ResolverContext>({
+  name: "RecentlySoldArtworkType",
+  fields: {
+    lowEstimateUSD: {
+      type: GraphQLString,
+    },
+    highEstimateUSD: {
+      type: GraphQLString,
+    },
+    priceRealized: {
+      type: GraphQLString,
+    },
+    artwork: {
+      type: ArtworkType,
+    },
+  },
+})
+
+export const RecentlySoldArtworks: GraphQLFieldConfig<void, ResolverContext> = {
+  args: pageable({}),
+  type: connectionDefinitions({ nodeType: RecentlySoldArtworkType })
+    .connectionType,
+  resolve: async (_, options, { artworksLoader }) => {
+    const { page, size } = convertConnectionArgsToGravityArgs(options)
+    const totalCount = recentlySoldArtworks.length
+    const ids = recentlySoldArtworks.map(({ artworkId }) => artworkId)
+
+    const response = await artworksLoader({ ids })
```

Comment:
**suggestion (non-blocking):** Since we're already hard-coding the list of recently sold artworks, why not also hard-code the artwork data that the rails need and skip this step?

Reply:
I would prefer to use fresh artwork information just in case since looks like the query correctly works with cache.

Reply:
[Gravity doesn't return deleted/unpublished artworks](https://github.com/artsy/gravity/blob/acf176e92bb053b11a9947db22554e33074238a6/app/api/v1/artworks_endpoint.rb#L195-L198), so I'm not sending such artworks to the client. this helps to prevent redirects to the artwork's deleted pages on artwork card click.

Reply:
Understood 👍 Thanks for adding "static" to the description to make this clear for future developers.